mailet/standard/src/main/java/org/apache/james/transport/mailets/RemoveMimeHeader.java

```diff
@@ -63,9 +66,25 @@ public String getMailetInfo() {
     @Override
     public void service(Mail mail) throws MessagingException {
         MimeMessage  message = mail.getMessage();
+        
+        // Global
         for (String header : headers) {
             message.removeHeader(header);
         }
+        
+        // Specific
```

Comment:
instead of comments, you could extract some meaningful methods, for example here `removeGlobalHeaders()` and `removeRecipientSpecificHeaders()`

Reply:
Are you sure you want that? We are not reusing these methods, so that is adding more depth to the stack and also, the code is harder to follow due to an extra redirection

Reply:
yes, it allows to have shorter methods, so it improves readability and maintenance